frontend/test/account-send.test.ts

```diff
@@ -0,0 +1,309 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
Looks great! Thanks for fuzz and parameterized tests and catching some additional bugs